src/Symfony/Bundle/FrameworkBundle/Resources/config/message.xml

```diff
@@ -9,7 +9,7 @@
 
         <!-- Bus -->
         <service id="message_bus" class="Symfony\Component\Message\MessageBus" public="true">
-            <argument type="collection" /> <!-- Middlewares -->
+            <argument type="tagged" tag="message_middleware" /> <!-- Middlewares -->
```

Comment:
This means that you can't register another bus with different middlewares. All of them will get injected here which is probably not what you want.

Reply:
Hence the "(now)" in the PR commit. It's an iteration until the last step (IMO) of the PR which is allowing the creation of multiple buses.